static/app/views/replays/deadRageClick/deadRageSelectorCards.tsx

```diff
@@ -0,0 +1,126 @@
+import {ComponentProps, ReactNode} from 'react';
+import styled from '@emotion/styled';
+import {Location} from 'history';
+
+import {LinkButton} from 'sentry/components/button';
+import {hydratedSelectorData} from 'sentry/components/replays/utils';
+import {IconShow} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import useDeadRageSelectors from 'sentry/utils/replays/hooks/useDeadRageSelectors';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
+import SelectorTable from 'sentry/views/replays/deadRageClick/selectorTable';
+
+function DeadRageSelectorCards() {
+  const location = useLocation();
+
+  return (
+    <SplitCardContainer>
+      <DeadClickTable location={location} />
+      <RageClickTable location={location} />
+    </SplitCardContainer>
+  );
+}
+
+function DeadClickTable({location}: {location: Location<any>}) {
+  const {isLoading, isError, data} = useDeadRageSelectors({
+    per_page: 3,
+    sort: '-count_dead_clicks',
+    cursor: undefined,
+  });
+
+  return (
+    <SelectorTable
+      data={hydratedSelectorData(data, 'count_dead_clicks')}
+      isError={isError}
+      isLoading={isLoading}
+      location={location}
+      clickCountColumn={{key: 'count_dead_clicks', name: 'dead clicks'}}
+      clickCountSortable={false}
+      title="Most Dead Clicks"
+      headerButtons={
+        <SearchButton
+          label={t('Show all')}
+          sort="-count_dead_clicks"
+          path="dead-clicks" // temporary; this might point to a tab on the replay index later
+        />
+      }
+      customHandleResize={() => {}}
+    />
+  );
+}
+
+function RageClickTable({location}: {location: Location<any>}) {
+  const {isLoading, isError, data} = useDeadRageSelectors({
+    per_page: 3,
+    sort: '-count_rage_clicks',
+    cursor: undefined,
+  });
+
+  return (
+    <SelectorTable
+      data={hydratedSelectorData(data, 'count_rage_clicks')}
+      isError={isError}
+      isLoading={isLoading}
+      location={location}
+      clickCountColumn={{key: 'count_rage_clicks', name: 'rage clicks'}}
+      clickCountSortable={false}
+      title="Most Rage Clicks"
+      headerButtons={
+        <SearchButton
+          label={t('Show all')}
+          sort="-count_rage_clicks"
+          path="rage-clicks" // temporary; this might point to a tab on the replay index later
```

Comment:
don't need the comment 😺 

When/if we want to use tabs what we'll do is add a tab component to the top of each page, and make sure the correct one is selected. maybe a refactor of the header bits to share some code, but it's not going to be a big lift and we'll be able to use the same urls we have now.

Issues does this, the urls are something like `/issues/:issueId/` for the details view, and then `/issues/:issueId/activity/`, `/issues/:issueId/attachments/`, `/issues/:issueId/replays/`, etc, etc.